src/api/user/service.js

```diff
@@ -35,4 +36,27 @@ export class UserService {
   logout() {
     this.currentUser = null;
   }
+
+  create(username, roles) {
+    return axios.post('backend://users', { username, roles: [roles] })
+      .then((response) => this.users = this.hydrateArray(response.data)); // todo remove this temporary fix.
```

Comment:
do we get an array as response? 
anyway, we could ad a static create method in the entity instead, an call UserEntity.create(response.data)